### ISSUE TITLE
ROHeatshields roughly twice as expensive as other shields

### ISSUE THREAD

**Issue by lpgagnon** [KSP-RO/RP-0#1853]
Stock 1m shield (RO.LEOHeatShield1m: 400 funds). 1m ROHeathshield with Mercury config: 800 funds

2m: 800 vs 1531

4m: 1500 vs 3601

A small amount of that comes from getting a bit more ablator, but this only accounts for a small fraction of the difference.

Interestingly, in the parts catalog, the nominal cost of the ROHeatshield part (which defaults to 1m) is 400 funds, matching the 1m stock shield.

**Comment by siimav** [comment]
I think most of the prefab parts got removed but there are still some capsule-specific ones that need double checking.

**Comment by NathanKell** [comment]
Yeah we can change around costs such that capsule shields are about what ROH would be, and the difference is applied to the capsule itself, or something.

**Comment by siimav** [comment]
Just confirmed that capsule-specific parts are still a lot less expensive than the ROHS equivalent.
